cmake/FindSTC.cmake

```
@@ -1,3 +1,3 @@
 include(CPM)
 
-CPMAddPackage(NAME stc GIT_REPOSITORY https://github.com/tylov/stc GIT_TAG v4.1.1 VERSION 4.1.1 OPTIONS "BUILD_TESTING OFF")
\ No newline at end of file
+CPMAddPackage(NAME stc GIT_REPOSITORY https://github.com/tylov/stc GIT_TAG v4.2 VERSION 4.2 OPTIONS "BUILD_TESTING OFF")
```

Repo has moved:  https://github.com/stclib/stc

```suggestion
CPMAddPackage(NAME stc GIT_REPOSITORY https://github.com/stclib/STC GIT_TAG v4.2 VERSION 4.2 OPTIONS "BUILD_TESTING OFF")
```